src/poliastro/plotting.py

```diff
@@ -137,6 +145,14 @@ def _project(self, rr):
         y = rr_proj.dot(self._frame[1])
         return x, y
 
+    def verify_attractor(self, orbit):
```

Comment:
Does it make sense to have a public method here? In `OrbitPlotter3D` we're doing `self.set_attractor` and then `self._redraw_attractor`. Can we do a similar thing here?

https://github.com/poliastro/poliastro/pull/332/files#diff-be441b39adc37d0e0a9887217a253560R330

Reply:
Yeah Sure! Now addressing this!

Reply:
Okay! So, basically, we would never merge these two classes. We would just remove the ambiguity for the user about the names of the methods and what they do... Am I right?